### PR TITLE
Update meta.yaml - New Version 0.3.0

### PR DESCRIPTION
**Pull request by rxm7706**
Bumped Python >=3.7
Added flint-core to build requirements
Removed attrs
Allowed click>8
Added recipe maintainer 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
[@chrisjsewell](https://github.com/chrisjsewell/)

Could you review.. builds are passing now

**Comment by chrisjsewell** [comment]
> [@chrisjsewell](https://github.com/chrisjsewell/)
> 
> Could you review.. builds are passing now

@choldgraf is the only maintainer of this

**Comment by choldgraf** [comment]
Hey all - i don't have time to maintain this repository. I am happy to add anybody else as a maintainer that would like to do so

**Comment by rxm7706** [comment]
@choldgraf  I'm just learning my way around feedstocks, you can add me, and I will help best I can.

**Comment by choldgraf** [comment]
Sounds good - thanks @rxm7706 for offering to help maintain these packages, it is much-appreciated :-) 

I wonder if @ocefpaf could help advise on how others can be added as admins on conda-forge packages, I've never done this before (I have almost no experience maintaining conda-forge projects as well)

**Comment by ocefpaf** [comment]
> I wonder if @ocefpaf could help advise on how others can be added as admins on conda-forge packages

You need to add yourself as a maintainers at the end of meta.yaml file. When merged you'll get commit rights here.

**Comment by choldgraf** [comment]
@rxm7706 wanna give that a shot? Here's the file: https://github.com/conda-forge/sphinx-external-toc-feedstock/blob/main/recipe/meta.yaml

**Comment by rxm7706** [comment]
thanks @ocefpaf 
@choldgraf I have added myself as a maintainer - if you merge this, it will upgrade to V 0.3.0 as well.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,3 +46,4 @@ about:
 extra:
   recipe-maintainers:
     - choldgraf
+    - rxm7706
```
Comment (ocefpaf) [inline recipe/meta.yaml:49]: :+1:

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Review by choldgraf — APPROVED** [review]
many thanks @rxm7706 - I'll merge this one in then!

**Pull request merged**